src/lamination/Laminator.sol

```diff
@@ -19,7 +19,11 @@ contract Laminator is ILaminator {
     /// @param proxyAddress The address of the proxy contract where the function call is pushed.
     /// @param callObjs The CallObject containing the function call details.
     /// @param sequenceNumber The sequence number assigned to the deferred function call.
-    event ProxyPushed(address indexed proxyAddress, CallObject[] callObjs, uint256 sequenceNumber);
+    /// @param selector code identifier for solvers to select relevant actions
+    /// @param data values to be used by solvers in serving the user objective
+    event ProxyPushed(
+        address indexed proxyAddress, CallObject[] callObjs, uint256 sequenceNumber, bytes indexed selector, bytes data
```

Comment:
Question about bytes data. What about the format? I was expecting an array of name - type - value triples.

Reply:
Serialization / deserialization between solidity and rust can become a problem unless we have a deserialization library. I'd still go with more certain format, array of name - type - value triple. For my understanding it would be more than enough.

Reply:
Let me try that without encoding, will create a struct.